package.json

```
@@ -6,7 +6,8 @@
   "scripts": {
     "ng": "ng",
     "compile": "gulp compile",
-    "build": "npm run build --ws",
+    "copy-icon-font": "npx ncp ./node_modules/@cdssnc/gcds-fonts/fonts/icons ./packages/web/src/assets/fonts/icons",
```

```suggestion
    "copy-icon-font": "ncp ./node_modules/@cdssnc/gcds-fonts/fonts/icons ./packages/web/src/assets/fonts/icons",
```

I think we can remove `npx` because we have `ncp` as a dev dependency, it would be nice to reduce the overhead for the build time